spec/features/medicaid_application_with_maximum_info_spec.rb

```diff
@@ -126,38 +126,28 @@
     on_pages "Income & Expense Amounts" do
       click_on "Next"
 
-      fill_in "step_employed_pay_quantities_0", with: 100
-      fill_in "step_employed_pay_quantities_1", with: 50
-      fill_in "step_employed_pay_quantities_2", with: 25
-
-      fill_in "step_employed_employer_names_0", with: "Acme Co."
-      fill_in "step_employed_employer_names_1", with: "CfA"
-      fill_in "step_employed_employer_names_2", with: "Cylinder"
-
-      select "Monthly", from: "step_employed_payment_frequency_0"
-      select "Weekly", from: "step_employed_payment_frequency_1"
-      select "Hourly", from: "step_employed_payment_frequency_2"
+      within(find("fieldset", text: "Job #1")) do
+        fill_in "Employer name", with: "CfA"
+        fill_in "Average amount / paycheck", with: 100
+        select "Weekly", from: "How often are you paid that amount?"
+      end
+
+      within(find("fieldset", text: "Job #2")) do
+        fill_in "Employer name", with: "Cylinder"
+        fill_in "Average amount / paycheck", with: 25
+        select "Hourly", from: "How often are you paid that amount?"
+      end
+
+      within(find("fieldset", text: "Job #3")) do
+        fill_in "Employer name", with: "Acme Co."
+        fill_in "Average amount / paycheck", with: 50
+        select "Monthly", from: "How often are you paid that amount?"
+      end
 
       fill_in "step_self_employed_monthly_income", with: 100
       fill_in "step_unemployment_income", with: 100
       click_on "Next"
 
-      click_back
```

Comment:
We want to note that in this PR we removed this part of the feature test that clicks the back button and ensured that data is pre-filled correctly. Our instinct was that this doesn't make sense in this feature test. Does that sound right? @luigi @jessieay @hartsick